DESCRIPTION

```diff
@@ -1,11 +1,12 @@
 Package: hySpc.read.spe
 Type: Package
 Version: 0.0.0.9000
-Title: File Import Functions for Spectra produced by Princeton Instruments spectrometers
+Title: File Import Functions for Spectra Produced by Princeton Instruments Spectrometers
 Authors@R: c(
     person("Claudia",   "Beleites",  role = c("aut"), email = "Claudia.Beleites@chemometrix.gmbh"),
     person("Sang",     "Truong", role = c("aut","cre"), email = "sttruong@stanford.edu"),
```

Comment:
1. If Sang will be the maintainer of this package (`cre`), then I'm not sure if I have to be included as an author . (`aut`). Contributor (`ctb`) would be OK for now.

2. THe order of authors (in my opinion) should go in this order (by the amount of contribution) Claudia and Roman, then Sang, then others.

Reply:
@sangttruong, when you update this file, I think, this portion of changes could be merged.

Reply:
It's good to go now @GegznaV